scipy/linalg/flapack.pyf.src

```diff
@@ -677,7 +677,7 @@ interface
      callstatement (*f2py_func)((lower?"L":"U"),&n,a,&lda,ipiv,work,&lwork,&info)
      callprotoargument char*,int*,<ctype>*,int*,int*,<ctype>*,int*,int*
 
-     integer optional,intent(in),check(lower==0||lower==1):: lower = 1
+     integer optional,intent(in),check(lower==0||lower==1):: lower = 0
```

Comment:
Can you please add `.... :: lwork = max(1, n)` to this one instead of `:: lwork = n` (line 685) and `hetrf` while we are at it? Otherwise `np.empty((0,0))` will cause segfaults.

Reply:
But then you can't query work-size?

Reply:
Or does explicit `-1` not go through `max`?

Reply:
You can still supply `-1` and it is allowed but if not supplied currently it defaults to `n` and if `n` is zero then `lwork` becomes 0. That in turn makes `work` a 0D array and that causes segfaults.